ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml

```diff
@@ -577,7 +577,7 @@ tests:
     workflow: openshift-e2e-libvirt-upi
 - as: ocp-e2e-ovn-ppc64le-powervs-original
   cluster: build01
-  cron: 0 0,6,12,18 * * *
+  cron: 0 12 * * *
```

Comment:
Do we want this job running at the same time as 4.15?

Reply:
Also, historically we've used the same timings across different regions and it's worked fine!

Reply:
Perfect!  Just checking :D